Sources/DolbyIORTSUIKit/Private/Views/StatsInfo/StatisticsInfoView.swift

```diff
@@ -43,8 +44,8 @@ struct StatisticsInfoView: View {
                 
                 ForEach(viewModel.data) { item in
                     HStack {
-                        Text(item.key).font(fontTable).frame(width: 170, alignment: .leading)
-                        Text(item.value).font(fontTable).frame(width: 170, alignment: .leading)
+                        Text(item.key).font(fontTable).foregroundColor(Color(theme.neutral200)).frame(width: 170, alignment: .leading)
+                        Text(item.value).font(fontTableValue).foregroundColor(Color(theme.onBackground)).bold().frame(width: 170, alignment: .leading)
```

Comment:
Do we need to use .bold() here as the font itself is bold ?

Reply:
Thanks a lot, sure you're right, addressed